### PR TITLE
Added folder PWGCF/FLOW/Forward for forward flow analysis

### PR DESCRIPTION
**Pull request by FrejaThoresen**

### CODE REVIEW

**Comment by rbertens** [comment]
+test

File: PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  //AliForwardNUATask* task = new AliForwardNUATask(name);
+
+  TString resName = "awesome";
+
+  task->fSettings.doNUA = doNUA;
+
+
+
+  if (task->fSettings.doNUA){
+      TFile* file = new TFile("/home/thoresen/Documents/PhD/Analysis/nua.root");
```
Comment (akubera) [inline PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C]: There are a couple instances of hard-coded paths. Maybe it's not a problem but I thought I'd point it out.

Comment (rbertens) [reply PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C]: Hi @FrejaThoresen , the PR looks good, but indeed could you work around the hardcoded paths that @akubera  pointed out (apart form this, there's a few other mentions of your home folder)? thanks, redmer

Comment (akubera) [reply PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C]: I'd suggest using an environment variable, say NUA_FILE, and use the [std::getenv](http://en.cppreference.com/w/cpp/utility/program/getenv) function to load the path. 
Maybe something like this:
```c++
  TString nua_filepath = std::getenv("NUA_FILE");
  if (!nua_filepath) {
    nua_filepath = "/home/thoresen/Documents/PhD/Analysis/nua.root"
    std::cerr << "Environment variable 'NUA_FILE' not found (this should be a path to nua.root).\n";
    std::cerr << "   Using default value: '" << nua_filepath << "'\n";
  }
  TFile *file = new TFile(nua_filepath);
```
Then you can use an `alien://path/to/nua.root` that can be used on the grid.

Comment (FrejaThoresen) [reply PWGCF/FLOW/Forward/macros/AddTaskForwardFlowRun2.C]: Thanks for the comments, I forgot about the local path. I will change it to a more generic setting.

**Comment by rbertens** [comment]
+1 @FrejaThoresen thanks for the changes

**Comment by mfasDa** [comment]
@FrejaThoresen @rbertens @akubera This commit breaks compilation on OS X:

DEBUG:AliPhysics:ali-master: /Users/markusfasel/alice/sw/SOURCES/AliPhysics/0_ROOT6/0/PWGCF/FLOW/Forward/AliForwardFlowRun2Task.cxx:153:71: error: non-constant-expression cannot be narrowed from type 'int' to 'Double_t' (aka 'double') in initializer list [-Wc++11-narrowing]
DEBUG:AliPhysics:ali-master:       Double_t xmax[5] = {10,fSettings.fZVtxAcceptanceUpEdge, 6, 100, fSettings.kSinphi1phi2phi3p+1};
DEBUG:AliPhysics:ali-master:                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
DEBUG:AliPhysics:ali-master: /Users/markusfasel/alice/sw/SOURCES/AliPhysics/0_ROOT6/0/PWGCF/FLOW/Forward/AliForwardFlowRun2Task.cxx:153:71: note: insert an explicit cast to silence this issue
DEBUG:AliPhysics:ali-master:       Double_t xmax[5] = {10,fSettings.fZVtxAcceptanceUpEdge, 6, 100, fSettings.kSinphi1phi2phi3p+1};
DEBUG:AliPhysics:ali-master:                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
DEBUG:AliPhysics:ali-master:                                                                       static_cast<Double_t>(       )
DEBUG:AliPhysics:ali-master: 1 error generated.
DEBUG:AliPhysics:ali-master: make[2]: *** [PWGCF/FLOW/Forward/CMakeFiles/PWGCFFlowForward.dir/AliForwardFlowRun2Task.cxx.o] Error 1
DEBUG:AliPhysics:ali-master: make[2]: *** Waiting for unfinished jobs....
DEBUG:AliPhysics:ali-master: [ 80%] Built target PWGLFQATasks
DEBUG:AliPhysics:ali-master: [ 80%] Generating libPWGEMCALtrigger.rootmap, G__PWGEMCALtrigger.cxx, G__PWGEMCALtrigger_rdict.pcm
DEBUG:AliPhysics:ali-master: /Users/markusfasel/alice/sw/SOURCES/AliPhysics/0_ROOT6/0/PWGCF/FLOW/Forward/AliForwardSecondariesTask.cxx:272:71: error: non-constant-expression cannot be narrowed from type 'int' to 'Double_t' (aka 'double') in initializer list [-Wc++11-narrowing]
DEBUG:AliPhysics:ali-master:       Double_t xmax[5] = {10,fSettings.fZVtxAcceptanceUpEdge, 6, 100, fSettings.kSinphi1phi2phi3p+1};
DEBUG:AliPhysics:ali-master:                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
DEBUG:AliPhysics:ali-master: /Users/markusfasel/alice/sw/SOURCES/AliPhysics/0_ROOT6/0/PWGCF/FLOW/Forward/AliForwardSecondariesTask.cxx:272:71: note: insert an explicit cast to silence this issue
DEBUG:AliPhysics:ali-master:       Double_t xmax[5] = {10,fSettings.fZVtxAcceptanceUpEdge, 6, 100, fSettings.kSinphi1phi2phi3p+1};
DEBUG:AliPhysics:ali-master:                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
DEBUG:AliPhysics:ali-master:                                                                       static_cast<Double_t>(       )
DEBUG:AliPhysics:ali-master: Scanning dependencies of target PWGLFresonances
DEBUG:AliPhysics:ali-master: 1 error generated.
DEBUG:AliPhysics:ali-master: make[2]: *** [PWGCF/FLOW/Forward/CMakeFiles/PWGCFFlowForward.dir/AliForwardSecondariesTask.cxx.o] Error 1
DEBUG:AliPhysics:ali-master: make[1]: *** [PWGCF/FLOW/Forward/CMakeFiles/PWGCFFlowForward.dir/all] Error 2
DEBUG:AliPhysics:ali-master: make[1]: *** Waiting for unfinished jobs....

Could you please have a look?

Thanks!

Cheers

Markus

**Comment by rbertens** [comment]
Hi @mfasDa , I've just opened #3609 that should mitigate this issue. I don't have a root6 build available for testing right now, could you let me know if you still encounter problems? Apologies for the disruption. 
Redmer

**Comment by FrejaThoresen** [comment]
Hi @rbertens and @mfasDa 
I can fix this by using static_cast, and commit again if preferred.
Cheers

**Comment by rbertens** [comment]
Hi @FrejaThoresen , it was  already fixed with PR #3609 yesterday, so no action is needed from your side. 
thanks,
redmer

**Pull request merged**